app/routes/circulars/circulars.server.ts

```diff
@@ -216,7 +217,11 @@ export async function search({
   try {
     searchResult = await client.search(searchBody)
   } catch (error) {
-    if ((error as typeof Error).toString().includes('Failed to parse query')) {
+    if (
+      (error as errors.ResponseError).body.error.root_cause[0].reason.includes(
+        'Failed to parse query'
+      )
```

Comment:
Please avoid using [type assertions](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions). Instead, actually test if the object has the required type.

```suggestion
      error instanceof errors.ResponseError && error.message.includes(
        'Failed to parse query'
      )
```